src/submission/exceptions/exceptions.ts

```diff
@@ -321,14 +323,23 @@ export async function getExceptionManifestRecords(
 		createProgramExceptions(programId),
 	);
 
+	const sortedFollowUpExceptions = entityPropertyException
+		? entityPropertyException.follow_up.sort(sortExceptionRecordsBySubmitterId)
+		: [];
+
+	const sortedSpecimenExceptions = entityPropertyException
+		? entityPropertyException.specimen.sort(sortExceptionRecordsBySubmitterId)
+		: [];
+
+	const sortedTreatmentExceptions = entityPropertyException
+		? entityPropertyException.treatment.sort(sortExceptionRecordsBySubmitterId)
+		: [];
+
```

Comment:
ternary check with the same condition `entityPropertyException` and the same falsy val `[]`
could be cleaned up a little with 
```ts
	const sortedTreatmentExceptions =
		entityPropertyException?.treatment.sort(sortExceptionRecordsBySubmitterId) || [];
```

Reply:
Thanks I knew this could be improved and the `?` just wasn't coming to me